packages/block-editor/src/components/link-control/style.scss

```diff
@@ -67,6 +67,12 @@
 	}
 }
 
+.block-editor-link-control__search-results-label {
+	padding: 15px 30px 0 30px;
+	display: block;
+	font-size: 1.1em;
```

Comment:
It seems like all the other `font-size` definitions are in `px`. Should this be changed to align with that?

Reply:
I want it to be based on a scale so it's "slightly larger" than whatever the base font size is for the component.